### ISSUE TITLE
Download link for ARM

### ISSUE THREAD

**Issue by ArMour85** [nim-lang/website#282]
Hello,

The latest version 1.4.8 of Nim exists with [ARM 64 and 32 bits](https://github.com/nim-lang/nightlies/releases/tag/2021-05-25-version-1-4-44e653a9314e1b8503f0fa4a8a34c3380b26fff3), a link to download these binaries in the Download page will be a good thing.

**Comment by ArMour85** [comment]
I'm raising the topic with this question. Also, do I need to make an issue in Nim's git to ARM releases to be uploaded to **https://nim-lang.org/download/**.

**Comment by narimiran** [comment]
I'll try to come up with some solution.

Should it be under this page: https://nim-lang.org/install_unix.html ?

**Comment by ArMour85** [comment]
Yes i think it’s the good page in the *Pre-built binaries for Linux* section.

There is also ARM for MacOS now but I don’t know if this version exists for the latest version of Nim. Maybe also adding a section on MacOS x64 binaries for the moment😊
